bower.json

```diff
@@ -4,10 +4,13 @@
     "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",
     "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",
     "main": [
-        "jquery.validate.unobtrusive.js",
-        "jquery.validate.unobtrusive.min.js"
+        "jquery.validate.unobtrusive.js"
+    ],
+    "ignore": [
+        "*.json",
```

Comment:
I’ve added an ignore for dot (.) files and folders. Not sure about bower.json, probably a feature ☺

From: Kiran Challa [mailto:notifications@github.com]
Sent: Saturday, 29 August 2015 5:38 AM
To: aspnet/jquery-validation-unobtrusive
Cc: David Poirier
Subject: Re: [jquery-validation-unobtrusive] Remove .min file from bower.json main section (#10)


In bower.json<https://github.com/aspnet/jquery-validation-unobtrusive/pull/10#discussion_r38236101>:

> @@ -4,10 +4,13 @@

>      "homepage": "https://github.com/aspnet/jquery-validation-unobtrusive",

>      "description": "Add-on to jQuery Validation to enable unobtrusive validation options in data-* attributes.",

>      "main": [

> -        "jquery.validate.unobtrusive.js",

> -        "jquery.validate.unobtrusive.min.js"

> +        "jquery.validate.unobtrusive.js"

> +    ],

> +    "ignore": [

> +        "*.json",

should add .gitignore..no? BTW I noticed that even though we try to ignore bower.json here it still gets installed by bower when installing a package.

Thanks! for updating the other repo too

—
Reply to this email directly or view it on GitHub<https://github.com/aspnet/jquery-validation-unobtrusive/pull/10/files#r38236101>.

______________________________________________________________________
Notice: This email and attachments (if any) is for the exclusive use of the addressee and may contain information that is privileged and confidential. Any unauthorised use of the contents of the email and/or attachments is expressly prohibited. If you are not the addressee of this email, please notify me immediately by email and then immediately destroy any electronic or paper copy of this email and/or attachments. Carsales.com Ltd and its related companies do not accept responsibility for the views expressed in the email or for the consequences of any computer viruses that may be transmitted with this email. This email is subject to copyright. No part of it should be reproduced, adapted or transmitted without the written consent of the copyright owner.